index.js

```diff
@@ -306,6 +306,12 @@
     };
   };
 
+  //  ValidDate :: Type
+  var ValidDate = $.NullaryType(
+    'sanctuary-def/ValidDate',
+    function(x) { return $$typeEq('Date')(x) && !isNaN(x.valueOf()); }
+  );
+
```

Comment:
Were we to define this after `$.Date` we could leverage `$.Date.test`:

```javascript
//  ValidDate :: Type
$.ValidDate = $.NullaryType(
  'sanctuary-def/ValidDate',
  function(x) { return $.Date.test(x) && !isNaN(x.valueOf()); }
);
```

This would make it very clear that `$.ValidDate` is `$.Date` sans `new Date(NaN)`.